docs/extensions.md

```diff
@@ -21,11 +21,14 @@ Extensions must include at least one schema file. In most cases, the extension w
 
 Repositories for [core extensions](http://standard.open-contracting.org/latest/en/extensions/#core-extensions) should have issue submissions disable by default and should direct developers to the [OCDS extension issues repository](https://github.com/open-contracting/ocds-extensions) to file issues. Best practice is to add that information to the _README.md_ file in every core extension using the following template:
 
-  > _&#35;&#35;&#35; Reporting issues_
-  >
-  > _By default, issues are disabled for individual OCDS core extensions. Instead, there is a &#91;ocds extension repository &#93;&#40;https&#58;&#47;&#47;github.com/open-contracting/ocds-extensions/&#41; to gather issues for all extensions in a single place._
-  >
-  > _If you have an issue to report, please submit it there. Make sure you indicate the appropriate extension following this format in the title: &#42;&#42;&#95;extension_name: issue title&#95;&#42;&#42;._
+```none
+
+### Reporting issues
+
+By default, issues are disabled for individual OCDS core extensions. Instead, there is a [ocds extension repository]https://github.com/open-contracting/ocds-extensions) to gather issues for all extensions in a single place._
+
+If you have an issue to report, please submit it there. Make sure you indicate the appropriate extension following this format in the title: **_extension_name: issue title_**;.
```

Comment:
Stray semi-colon at the end :)

Reply:
I just changed that.

Reply:
Odd, GitHub says the comment is out-dated, but I still see `title_**;.` after refresh.

Reply:
I think that is my fault, as I'm amending commits (which is _kind of_ OK if you are sure nobody pulled the branch so far) and that confuses GitHub. If you look in "Files changed" you'll see the text has been fixed.